api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/handler/GaugeHandler.java

```diff
@@ -321,82 +321,83 @@ public void findGaugeData(
                 return;
             }
 
-            if (limit != null) {
```

Comment:
Do you mind reverting this method to the original prior to this PR? I think this is mostly formatting, but this method needs to be unchanged.